src/content/events/2023/training-sept-2023/wgc.md

```diff
@@ -22,7 +22,7 @@ If you have any questions about this event please reach out to [Priyanka Surana]
 
 # Local Schedule
 
-This full-day event will run from **9.30 am - 5.00 pm BST**.
+This full-day event will run from **9.30 am - 5.00 pm+01:00**.
```

Comment:
Is this a copypasta error? I assume that free text doesn't make any difference..

Reply:
yep, sorry, thought I caught them all